src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const cleanedSource = sourceString.trim().replace(/;\s*$/, '');
+
+  const declarations = cleanedSource
+    .split(';')
+    .filter((declaration) => declaration.trim() !== '');
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable.